### DISCUSSION TITLE
Synchronization of two or more  devices

### DISCUSSION THREAD

**Discussion by Massimo65** [arendst/Tasmota#13875] [Q&A] [answered]
Good morning

I need to synchronize two tasmota devices. specifically, if I activate the relay on one device, I need it to activate on the other as well. can you help me?
I tried to follow the guides but it doesn't seem to work
Current version 10.

thk

**Comment by pcdiem** [comment, marked as answer]
If the devices are in the same Ethernet broadcast domain, you could put the devices in a device group. On both devices, enter Backlog DevGroupName Group1; SO85 1
Relay transitions on one devices will be forwarded to the other device(s) in the group via UDP multicast/unicast.

**Comment by Massimo65** [comment]
thank you 1000
